js/layout-javascript/news-feed-code.js

```diff
@@ -2393,6 +2393,7 @@ DynamicList.prototype.saveComment = function(entryId, commentId, value) {
       return instance.update({
         settings: commentData.data.settings
       }, {
+        id: commentId,
```

Comment:
The `where` clause  beneath would be unnecessary if we pass the `id` property.

Reply:
fixed